geomet_data_registry/layer/model_gem_global.py

```diff
@@ -111,19 +111,19 @@ def identify(self, filepath):
             interval = re.sub('[^0-9]', '', interval)
 
             fh = file_pattern_info['fh']
-            if int(fh) in range(int(begin), int(end), int(interval)):
-                feature_dict = {
-                    'layer_name': layer_name,
-                    'filepath': filepath,
-                    'identifier': identifier,
-                    'reference_datetime': reference_datetime,
-                    'forecast_hour_datetime': forecast_hour_datetime,
-                    'member': member,
-                    'model': self.model,
```

Comment:
This is needed and was removed in your merge request. 

Can you please add it ?

Reply:
Added! Sorry about that.